sklearn/model_selection/_split.py

```diff
@@ -1799,6 +1807,12 @@ class GroupShuffleSplit(ShuffleSplit):
     ...     print("TRAIN:", train_idx, "TEST:", test_idx)
     TRAIN: [2 3 4 5 6 7] TEST: [0 1]
     TRAIN: [0 1 5 6 7] TEST: [2 3 4]
+
+    See Also
+    --------
+    ShuffleSplit : Shuffles samples to create independent test/train sets.
+
+    LeavePGroupsOut : Test set leaves all possible subsets of ``p`` groups out.
```

Comment:
I think this is the behavior:

```suggestion
    LeavePGroupsOut : Train set leaves out all possible subsets of ``p`` groups.
```

For example:

```python
from sklearn.model_selection import LeavePGroupsOut

X = np.arange(6)
y = [1, 1, 1, 2, 2, 2]
groups = [1, 1, 2, 2, 3, 3]
lpgo = LeavePGroupsOut(n_groups=2)
for train, test in lpgo.split(X, y, groups=groups):
    print("%s %s" % (train, test))
# [4 5] [0 1 2 3]
# [2 3] [0 1 4 5]
# [0 1] [2 3 4 5]
```

In the first split, groups 1 and 2 are "left out" and go into the test set. Only group 3 is in the train set.

Reply:
Yes, got confused.